tests/Feature/BookTest.php

```diff
@@ -79,4 +79,41 @@ public function test_getAllBooks_success(): void
             })
         ;
     }
+
+    public function test_getAllBooks_genre(): void
+    {
+        Book::factory()->count(10)->create();
+
+        $response = $this->get('/api/books?genre');
```

Comment:
You should put a genre id here, like ?genre=1

Reply:
we tested this before and the number of genres would change each time, will try again tho